### PR TITLE
Comments on valuing money and negotiating

### PR DESCRIPTION
**Pull request by rudi-c**

### CODE REVIEW

File: docs/3-After-The-Offer/salary_and_negotiating.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
 Also, if you have another offer that's offering more money, you can let the other companies know that - sometimes they will match it!
 
 When I ask for more money, I not to make it seem like it's a strong deciding factor for me (because it's not) but that it's just something I'm thinking about.
 
 It's better to talk about salary on the phone (I know, phone calls suck - but this is how people do it and it'll probably give you the best chance. Write a script, it'll help)
 
-Check out data about salaries:
+### How much is a dollar worth?
+
+Rudi:
```
Comment (evykassirer) [inline docs/3-After-The-Offer/salary_and_negotiating.md]: maybe: "Some advice from [Rudi](http://digitalfreepen.com/about.html)"

File: docs/3-After-The-Offer/salary_and_negotiating.md [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+
+And maybe it is for you. Maybe you are struggling to pay off your living expenses and would like to support your family. Do what you need to do.
+
+For everyone else, keep in mind that you are doing an internship to learn, not to get rich. This isn't about how evil money is and all that. It's just that given that you are only working 3-4 months, the difference between your offers will be quite negligible in the long run in most cases. On the other hand, doing to a place with better learning opportunities will often unlock more options for future internships and full-time employment, some of which will pay more. And at that point, the difference will start to be significant, because you'll be working at that place for years. So even if money really is your end goal, think about it over the long run.
+
+Also, most humans are quite bad at estimating what will make them happy. You should read this [research on money and happiness](https://scholar.harvard.edu/files/danielgilbert/files/if-money-doesnt-make-you-happy.nov-12-20101.pdf).
```
Comment (evykassirer) [inline docs/3-After-The-Offer/salary_and_negotiating.md]: I shy away from "should"s - again, the feeling of having all these things I need to do to be prepared for internships, and making it more of a list of things to explore than things you have to do.

> Also, most humans are quite bad at estimating what will make them happy. I recommend [this read]

File: docs/3-After-The-Offer/salary_and_negotiating.md [hunk 3]
```diff
@@ -1,18 +1,56 @@
...
+3. They can hire someone just as good as you for the same amount of money
+4. They want all their interns to be paid the same to avoid resentment
+
+You should try to figure out what constraints they have to use the most effective approach in negotiating.
+
+It's difficult to do anything about (1) and (2). If you can demonstrate that you are significantly more productive, you might be able to make an economic argument that they'll get so much more work out of you that it's worth the slight increase in cost, but it will be hard.
+
+Case (3) will often be true at well-known, competitive companies. If you don't have a truly extraordinary accomplishment, negotiating might not be seen well.
+
+As a rule of thumb, if internship salary is publicly available for a company, (4) is likely to apply. You'll have a better chance of negotiating at smaller companies with fewer interns. Your negotiation is more likely to be successful if you can give them a story that would justify the salary disparity if other interns were to ever find out. This could include: "you have significantly more work experience", "you are senior", "you are a return intern".
```
Comment (evykassirer) [inline docs/3-After-The-Offer/salary_and_negotiating.md]: can also include "I was paid more at my last internship" or "my other offer is offering me more"

**Comment by rudi-c** [comment]
Changes:
- Small fixes

**Comment by rudi-c** [comment]
Fixed, forgot that I talked about company size in another section.

**Pull request merged**